### PR TITLE
Serve Google fonts locally instead of referencing Google servers.

### LINKED ISSUE #688: Remove dependence on Google Fonts (or make optional)
Using Google Fonts is a privacy issue since it executes code in your app that also tracks users. Here's some more background on the subject: http://fontfeed.com/archives/google-webfonts-the-spy-inside/

And a project to self-host Google Fonts (license permitting of course): http://ranf.tl/2014/12/23/self-hosting-google-web-fonts/

### PR DESCRIPTION
**Pull request by rwakida**
Fix #688.  Based on issue #688 advice, used webfont-dl to download roboto font and
modified base stylesheet to reference roboto locally instead of
referencing Google hosted version.

Used webfont-dl with url (originally declared in base.less.)
http://fonts.googleapis.com/css?family=Roboto:300,400,500,600,700&subset=latin,cyrillic-ext,greek-ext,greek,vietnamese,latin-ext,cyrillic
to download Roboto font to local directory. The url requests 300,400,500,600, and 700, but it looks like only 300,400,500, and 700 are supported.

Tested using Firefox.  Removed external url @import, cleared cache and saw non-roboto font displayed.  Also did not see external network requests for the font.  Added @import for local_font.import.less that referenced local fonts, restarted, and saw roboto font displayed.  

without referencing downloaded fonts:
no_import_fonts

with reference to downloaded fonts:
with_import_fonts

I tried testing with Chrome, but it always showed Roboto font even if I cleared/disabled cache.  The only way I could confirm is by checking the network requests thought the web console.

### CODE REVIEW

File: client/stylesheets/utils/_fonts.import.less [hunk 1]
```diff
@@ -1,72 +1,47 @@
...
-		 url('/fonts/robotocondensed-light-webfont.ttf') format('truetype'),
-		 url('/fonts/robotocondensed-light-webfont.svg#roboto_condensedlight') format('svg');
-	font-weight: 300;
-	font-style: normal;
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 300;
+  src: local("Roboto Light"), 
+    local("Roboto-Light"), 
+    url("/fonts/roboto-23d439d23da3eaa8144f02e19043fab2b3f19a33.woff2") format("woff2"),
```
Comment (rwakida) [inline client/stylesheets/utils/_fonts.import.less]: webfont-dl generates sha1-hash for filenames.  I couldn't find a way to generate user friendly names.

File: client/stylesheets/utils/_fonts.import.less [hunk 2]
```diff
@@ -1,72 +1,47 @@
 @font-face {
```
Comment (rwakida) [inline client/stylesheets/utils/_fonts.import.less]: This file contained Roboto font declarations, but was never referenced so I assumed the contents could be replaced.  I replaced the values using webfont-dl generated css.

**Comment by mhkeller** [comment]
I pulled the latest version today, restarted the server and I'm still seeing the request to fonts.googleapis.com and fonts.gstatic.com 

screen shot 2015-10-02 at 4 10 56 pm

**Comment by rwakida** [comment]
Hi @mhkeller, I tested using the latest from master and didn't see a request in privacy badger. Can another @RocketChat/owners please test.  
screenshot

**Comment by rwakida** [comment]
grep shows rocketchat-livechat package still references google fonts, but livechat is not a part of the main chat app.

**Comment by mhkeller** [comment]
Hmm is there something else I should be doing besides pulling the latest from github, running docker-compose stop, docker-compose start?

**Comment by geekgonecrazy** [comment]
@rwakida @mhkeller that package is disabled by default too.  

@mhkeller are you using the latest docker image?

**Comment by mhkeller** [comment]
ah if `git pull` isn't updating the docker image then that will surely do it. I'm not at that terminal currently but do i do `docker pull rocketchat/rocket.chat`?

**Comment by geekgonecrazy** [comment]
Correct.  Then Docker-compose up -d should relaunch the RocketChat image

**Comment by mhkeller** [comment]
Got it. That was it.

**Comment by geekgonecrazy** [comment]
Awesome!  Glad to hear!  :+1:

**Comment by rwakida** [comment]
thanks @geekgonecrazy :smile:

**Comment by geekgonecrazy** [comment]
@rwakida absolutely :grin:

**Pull request merged**